src/Views/ArticlesListView.jsx

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+import ArticlesList from '../Components/ArticlesList'
+
+
+export default function ArticlesListView() {
+    const path = 'https://nc-news-w53u.onrender.com/api/articles'
+    const [articles, setArticles] = useState([])
+    const dateConfig = {
+        year: '2-digit',
+        month: 'long',
+        day: '2-digit'
+    }
+    
+    const todaysDate = new Date().toLocaleDateString(undefined, dateConfig)
+
+    useEffect(() => {
+        axios.get(path)
```

Comment:
Rather than calling axios.get here, this should really be packed away in a separate function in an `api.js` file, which we can then just import 👍